src/inputs/labeled-inputs.scss

```diff
@@ -169,6 +167,45 @@
   }
 }
 
+/// Wrapping
+@mixin iui-input-label-wrapper {
+  @include iui-input-label-styling;
+  grid-area: label;
+  align-self: center;
+}
+
+/// Non-wrapping
+@mixin iui-input-label {
+  @include iui-input-label-styling;
+  display: block;
+  width: fit-content;
+  cursor: default;
+
+  @at-root {
+    label#{&} {
+      cursor: pointer;
+
+      &.iui-disabled {
+        cursor: not-allowed;
+      }
+    }
+  }
```

Comment:
Do we ever want to show a `default` cursor? I think this class should always be applied on a `label` element, so you could simplify the rules
```suggestion
  cursor: pointer;

  &.iui-disabled {
    cursor: not-allowed;
  }
```

Reply:
The default cursor is used if applied to a `<div>` and not a `<label>` like when we show a label for a group of checkboxes.

Reply:
I'm not so sure, because if I place the inline checkbox / radio as it is currently we end up with something like this:
![Screen Shot 2021-10-19 at 5 06 00 PM](https://user-images.githubusercontent.com/849817/137990525-eb011f2d-08d8-42e2-81d8-354cd4d7e830.png)

We would want something more like this:
![Screen Shot 2021-10-19 at 5 07 32 PM](https://user-images.githubusercontent.com/849817/137990720-58353a9f-0587-4c76-b2dc-e31699f578d0.png)

Reply:
And if I place the radio component outside of the grid, the labels don't align:
![Screen Shot 2021-10-19 at 5 08 36 PM](https://user-images.githubusercontent.com/849817/137990836-bcffd50f-7264-46d8-910e-5c881d0531da.png)